LibraryApp/src/app/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+
+import Book from './components/Book';
+import * as books from '../books.json';
```

Comment:
As this may be temporary, add a `TODO`. Try not to import `*`

Reply:
I tried not to mutate the books.json, that's why I don't do an export in that file and needed to import as *

Another way was to use require('route/book.json')... but it made the content of the file an object and not an array as I needed :/

Reply:
I correct myself... The way that it the content of the file is imported as an array is the second way, with require :sweat_smile: